defs/bugs.yaml

```diff
@@ -102,3 +102,9 @@ rabbitmq:
         restart or restarting them simultaneously will not work. See bug for
         more detail.
       path: 'var/log/rabbitmq/rabbit@*.log'
+    'rabbitmq/rabbitmq-server#2045':
```

Comment:
This should be an LP bug id. I see your other comment about adding multiple search expressions for the same bug and yes that is already possible e.g.

123456:
  expr:
    - 'expr1'
    - 'expr2'

Reply:
you've got two options;

1. open a bug and use that here
2. move this to events.yaml which would allow you to pull some information e.g. frequency

Reply:
I moved everything to `events.yaml`. Please have another look.